subprojects/execution/src/main/java/org/gradle/internal/execution/steps/ValidateStep.java

```diff
@@ -47,6 +47,7 @@
 
 public class ValidateStep<R extends Result> implements Step<AfterPreviousExecutionContext, R> {
     private static final Logger LOGGER = LoggerFactory.getLogger(ValidateStep.class);
+    private static final String MAX_NB_OF_ERRORS = "org.gradle.internal.max.validation.errors";
```

Comment:
Instead of making this configurable for tests, shall we maybe remove the limit altogether?

Reply:
We probably should, but I don't want to make it as part of this work, though.

Reply:
👍